packages/miniflare/test/plugins/kv/index.spec.ts

```diff
@@ -146,7 +146,35 @@ test("bulk get: check max keys", async (t) => {
 	try {
 		await kv.get(keyArray);
 	} catch (error: any) {
-		t.is(error.message, "KV GET_BULK failed: 400 Bad Request");
+		t.is(
+			error.message,
+			"KV GET_BULK failed: 400 You can request a maximum of 100 keys"
+		);
+	}
+});
+
+test("bulk get: check minimum keys", async (t) => {
+	const { kv } = t.context;
+	try {
+		await kv.get([]);
+	} catch (error: any) {
+		t.is(
+			error.message,
+			"KV GET_BULK failed: 400 You must request a minimum of 1 key"
+		);
+	}
+});
+
+test("bulk get: invalid type", async (t) => {
+	const { kv } = t.context;
+	try {
+		// @ts-ignore
+		await kv.get(["key"], {type: "invalid"});
```

Comment:
Nit: I wonder if this would be cleaner/clearer:
```suggestion
		await kv.get(["key"], { type: "invalid" as "json" });
```

Reply:
I did not think of that, thanks! Always better to not ignore ts things :)